docs/pages/modules/use-standalone-expo-module-in-your-project.mdx

```diff
@@ -5,7 +5,7 @@ description: Learn how to use a standalone module created with create-expo-modul
 
 import { Terminal } from '~/ui/components/Snippet';
 
-**The recommended way to create an Expo module** inside an existing application is described in the [Expo Modules API: Get started](/modules/get-started/) guide. This page walkthrough two additional methods of using a module created with `create-expo-module` inside an existing project:
+**The recommended way to create an Expo module** inside an existing application is described in the [Expo Modules API: Get started](/modules/get-started/) guide. This page walkthrough outlines two additional methods of using a module created with `create-expo-module` inside an existing project:
```

Comment:
```suggestion
**The recommended way to create an Expo module** inside an existing application is described in the [Expo Modules API: Get started](/modules/get-started/) guide. This page outlines two additional methods of using a module created with `create-expo-module` inside an existing project:
```

Both "walkthrough" and "outlines" kind of mean the same in the above context. We can just remove one.